app/pages/sections/funding-sources/ui-tests/general.ui.test.js

```diff
@@ -71,6 +71,24 @@ test('should render the description', async (t) => {
     .expect(await extractInnerText(description)).eql(content.description);
 });
 
+test('should render a selectFundingSource question as radio button options', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const selectFundingSourceRadioOptions = Selector('[data-test-id="question-selectFundingSource"]');
+
+  await t
+    .expect(selectFundingSourceRadioOptions.exists).ok()
```

Comment:
you can remove this check. If it doesn't exist the check underneath it will fail anyway